front/src/views/recovery-password/RecoveryPassword.css

```diff
@@ -0,0 +1,24 @@
+.MyInput{
```

Comment:
.input-email, .input-firstPass, .input-secondPass
these are three classes that have the same styles but are set in different places

Reply:
at first, .input-first-pass, .input-second-pass
pass - is password or what? i don't understand. did you read article from wiki? about ninja-code? 

is it only for your input? might this one could be place into common rules for inputs? if it is specific only for your view it could be .recovery-password-input. Also, please is it only one rule like justify: center? if  yes it could be .justify-content-center

Reply:
i fix it